wiz-admission-controller/templates/_helpers.tpl

```diff
@@ -189,3 +214,29 @@ Use for debug purpose only.
 {{- define "helpers.var_dump" -}}
 {{- . | mustToPrettyJson | printf "\nThe JSON output of the dumped var is: \n%s" | fail }}
 {{- end -}}
+
+{{- define "wiz-admission-controller.resources" -}}
+{{- if not .Values.hpa.enabled }}
+{{- if hasKey .Values "resources" }}
+{{- toYaml .Values.resources }}
+{{- else -}}
+{}
+{{- end -}}
+{{- else }}
+{{- if hasKey .Values "resources" }}
+{{- toYaml .Values.resources }}
+{{- else -}}
+requests:
+    cpu: 500m
+    memory: 300Mi
+{{- end -}}
+{{- end -}}
+{{- end -}}
```

Comment:
A minor change that IMO will make it more readable:
```
{{- define "wiz-admission-controller.resources" -}}
{{- if hasKey .Values "resources" }}
{{- toYaml .Values.resources }}
{{- else -}}
{{- if not .Values.hpa.enabled }}
{}
{{- else }}
requests:
    cpu: 500m
    memory: 300Mi
{{- end -}}
{{- end -}}
{{- end -}}
```

Reply:
I hope I didn't miss anything, please double check this 🙏

Reply:
This is way better, thanks!